Modules/Microsoft365DSC/Examples/Resources/IntuneDeviceCompliancePolicyAndroidDeviceOwner/1-NewDeviceCompliancePolicyAndroidDeviceOwner.ps1

```diff
@@ -13,33 +13,33 @@ Configuration Example
 
     node localhost
     {
-        IntuneDeviceCompliancePolicyAndroidDeviceOwner 'ConfigureAndroidDeviceCompliancePolicyOwner'
```

Comment:
Let's keep the Name in single quotes please. We recently revamped all examples to match this new pattern. Thanks

Reply:
Changed it to the required format and updated the ChangeLog